Build Fails on GDK_WINDOW_XID

<!--
HELP US TO HELP YOU BY PROVIDING THE FOLLOWING INFORMATION
-->

<!--
Information about WebKit Bugs:
Luakit is a built around the WebKit rendering engine. Some issues may therefore not be luakit related. Before opening a bug report here, please test if your issue also happens in epiphany (the gnome browser), which uses the same engine. If the issue happens there too, it is not a luakit bug.
-->

## I'm submitting a…
<!-- Bug Report / Feature Request / Other (please describe) -->
Bug Report
## Current Behavior
<!-- Describe the current behavior -->
When I run `make` on luakit it was looking for GDK_WINDOW_XID but could not find it

```
widgets/window.c:104:28: warning: implicit declaration of function ‘GDK_WINDOW_XID’; did you mean ‘GDK_WINDOW’? [-Wimplicit-function-declaration]
         lua_pushinteger(L, GDK_WINDOW_XID(
                            ^~~~~~~~~~~~~~
                            GDK_WINDOW
```
## Expected Behavior
<!-- Describe the expected/desired behavior -->
it should build sucessfully

## Reproduction Instructions
<!-- How can a developer reproduce the issue? Please explain step by step. -->
make
## Environment
<!-- Please provide information about your OS -->
Linux Distribution & Version:
linux 4.20.12, Archlinux
Output of `luakit --version`:
current head on git `209bdeb776858656391ad139ab41d31907caf4b4`

Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS

```
luakit/widgets/window.c:104: undefined reference to `GDK_WINDOW_XID'
luakit/widgets/window.c:113: undefined reference to `GDK_WINDOW_XID'
collect2: error: ld returned 1 exit status
Makefile:69: recipe for target 'luakit' failed
make: *** [luakit] Error 1
```

--- a/widgets/window.c
+++ b/widgets/window.c
@@ -101,7 +101,7 @@ luaH_window_index(lua_State *L, widget_t *w, luakit_token_t token)
 
 # ifdef GDK_WINDOWING_X11
       case L_TK_ROOT_WIN_XID:
-        lua_pushinteger(L, GDK_WINDOW_XID(
+        lua_pushinteger(L, GDK_WINDOW(
 #  if GTK_CHECK_VERSION(3,12,0)
                 gdk_screen_get_root_window(gtk_widget_get_screen(GTK_WIDGET(d->win)))
 #  else
@@ -110,7 +110,7 @@ luaH_window_index(lua_State *L, widget_t *w, luakit_token_t token)
         ));
         return 1;
 
-      PI_CASE(WIN_XID, GDK_WINDOW_XID(gtk_widget_get_window(GTK_WIDGET(d->win))));
+      PI_CASE(WIN_XID, GDK_WINDOW(gtk_widget_get_window(GTK_WIDGET(d->win))));
 # endif
 
       case L_TK_SCREEN:

